src/pages/home/Home.tsx

```diff
@@ -2,38 +2,58 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 
 import { makeStyles } from '@material-ui/core/styles';
+import { Grid, Typography } from '@material-ui/core';
 
-import { topBarHeight } from '../../features/frame/components/topbar/TopBar';
-
-const useStyles = makeStyles(() => ({
+const useStyles = makeStyles((theme) => ({
   root: {
     width: '100%',
-    display: 'flex',
-    flexDirection: 'column',
-    alignItems: 'center',
-    minWidth: '420px',
-    marginTop: 120,
   },
   rectangle67: {
-    display: 'flex',
-    justifyContent: 'flex-end',
-    position: 'absolute',
     width: '100%',
-    height: '534px',
-    left: '0px',
-    top: topBarHeight,
+    padding: 24,
     background: '#1282B2',
   },
+  content: {
+    padding: 24,
+    [theme.breakpoints.down('sm')]: {
+      padding: 8,
+    },
+    marginBottom: 32,
+  },
   wateringSunflower: {
-    position: 'absolute',
-    width: '374px',
-    height: '392.01px',
-    right: '211px',
-    top: '118px',
+    maxWidth: '100%',
+  },
+  title: {
+    color: '#ffffff',
+    textAlign: 'right',
+    fontSize: '400%',
```

Comment:
With the custom font, the font size is too big on medium screens:
![image](https://user-images.githubusercontent.com/4726554/132465948-7c315d95-b5fb-40d6-8980-4cc0348ac9eb.png)
```suggestion
    fontSize: '300%',
```

Reply:
Indeed. Another compromise to solve the case when the sidebar is open could be:
```diff
@@ -26,7 +26,7 @@ const useStyles = makeStyles((theme) => ({
   title: {
     color: '#ffffff',
     textAlign: 'right',
-    fontSize: '400%',
+    fontSize: '300%',
     [theme.breakpoints.down('sm')]: {
       textAlign: 'center',
       fontSize: '200%',
@@ -131,12 +131,12 @@ const Home = () => {
         <Grid xs={12}>
           <Navigation />
         </Grid>
-        <Grid xs={12} md={6} className={classes.titleContainer}>
+        <Grid xs={12} md={7} className={classes.titleContainer}>
           <Typography variant="h1" className={classes.title}>
             Collaborative content Recommendations
           </Typography>
         </Grid>
-        <Grid xs={12} md={6} className={classes.imageContainer}>
+        <Grid xs={12} md={5} className={classes.imageContainer}>
           <img src="/svg/Watering.svg" className={classes.wateringSunflower} />
         </Grid>
       </Grid>
```

Reply:
I fixed this by using 400% for xl, 300% md and sm, and 200% for xs